cfgov/jinja2/v1/_includes/molecules/form-field-with-button.html

```diff
@@ -27,24 +27,46 @@
 
    ========================================================================== #}
 
-{% macro render(settings) %}
-<div class="m-form-field-with-button">
-    <div class="form-group">
-        <label for="{{ settings.field.name }}">
-            <b>{{ settings.field.label }}</b>
-            {% if settings.field.required %}
-                (required)
-            {% endif %}
-        </label>
-        <input id="{{ (settings.field.id or settings.field.name) ~
-                   '_' ~ range(1, 100) | random }}"
-               type="{{ settings.field.type }}"
-               name="{{ settings.field.name }}"
-               placeholder="{{ settings.field.placeholder }}"
-               class="m-form-field-with-button_field"
-               {{ settings.field.attributes or '' }}>
+{% if value %}
+    <div class="m-form-field-with-button">
+        <div class="form-group">
+            <label for="{{ value.name }}">
+                <b>{{ value.label }}</b>
+                {% if value.required %}
+                    (required)
+                {% endif %}
+            </label>
+            <input id="{{ (value.id or value.name) ~
+                    '_' ~ range(1, 100) | random }}"
```

Comment:
Because of this _random-number, the input `id` doesn't match the label and we're getting an accessibility error. This should just be `<input id="{{ (value.id or value.name)">` and there should be something to prevent an id from being used more than once.